pkg/monitor/sqsevent/ec2-state-change-event.go

```diff
@@ -75,9 +75,6 @@ func (m SQSMonitor) ec2StateChangeToInterruptionEvent(event EventBridgeEvent, me
 		InstanceID:           ec2StateChangeDetail.InstanceID,
 		Description:          fmt.Sprintf("EC2 State Change event received. Instance went into %s at %s \n", ec2StateChangeDetail.State, event.getTime()),
 	}
-	if err != nil {
```

Comment:
Adding this error handling here made the `ec2-state-change-sqs-test` test fail. Not sure if swallowing this error is the way to go but this is out of the scope of this PR. I'm just replacing `err` with `_` for the sake of the linter but IMO this should be revisited.

Valid also for `pkg/monitor/sqsevent/rebalance-recommendation-event.go` and `pkg/monitor/sqsevent/spot-itn-event.go`